metron-platform/metron-elasticsearch/src/main/java/org/apache/metron/elasticsearch/dao/ElasticsearchDao.java

```diff
@@ -234,26 +366,43 @@ public synchronized void init(AccessConfig config) {
     if(this.client == null) {
       this.client = ElasticsearchUtils.getClient(config.getGlobalConfigSupplier().get(), config.getOptionalSettings());
       this.accessConfig = config;
+      this.columnMetadataDao = new ElasticsearchColumnMetadataDao(this.client.admin(), Collections.singletonList(".kibana"));
```

Comment:
While I agree with the inclusion of `.kibana` here, what are the consequences for those users with other, non-metron, indices?  Should we allow users to specify this list in the REST application.yml?

Reply:
Yes, I agree with your feedback.  I was just trying to refactor the column metadata logic and minimize other changes.  So in this case `.kibana` was already hard-coded in ElasticsearchDao.

I would be totally open to making this improvement though.  I was just trying to walk the line of how much should I change when refactoring?  Considering that, what do you think?

Reply:
Ok, man I have been really bad about reading and comprehension today.  I see in your review comment feedback that you already addressed this point. Sorry.

I think having this hardcoded is just not good.  The change seems simple enough.  I can probably take care of this no problem.

Reply:
I'm almost always in favor of having hard coded things passed in via config files.  It gets us out of jams and almost every time I've convinced myself that it's not necessary, it totally is.  How convinced are you that we are only ever going to need one ignored index?

If you have doubts, then I'd probably add something to the `application.yml` which gets set in the AccessConfig object when we set up the index in the IndexConfig from `metron-rest`.

Reply:
Given that it's a pretty simple change and requires no new machinery to added, I think it is probably a good fit to add to this PR, but I might be on the wrong side of that line more than I'm on the right side, so take my word with a grain of salt. ;)

Reply:
The latest has the change that makes the ignore indices configurable.